### ISSUE TITLE
Support for reddit albums/carousels

### ISSUE THREAD

**Issue by alixsep** [samirelanduk/reddit-save#12]
Hi,
Thanks for the great repo! I'm here to give some suggestion for improvement, reddit-save doesn't currently support downloading media from reddit posts that contain multiple images, like one of my posts: [Windows 11 Rose Wallpapers with JP flavor](https://www.reddit.com/r/Windows11/comments/ogwqlx/windows_11_rose_wallpapers_with_jp_flavor/)

<div align="center">

![image](https://user-images.githubusercontent.com/59757582/204180321-02e6fd5c-16b3-410f-b5cc-53320a49e81b.png)
(No media after downloading with reddit-save)

</div>

We'd be happy to get support!
Much thanks!
Alixsep

**Comment by samirelanduk** [comment]
I agree this would be a great feature - imgur albums have the same problem (see #1). I don't have the time to add this right now, but I can look at it again in the new year.
